components/sysutils/borgbackup/borg-34.p5m

```diff
@@ -98,3 +98,4 @@ file path=usr/lib/python3.4/vendor-packages/borgbackup-$(COMPONENT_VERSION)-py3.
 file path=usr/lib/python3.4/vendor-packages/borgbackup-$(COMPONENT_VERSION)-py3.4.egg-info/not-zip-safe
 file path=usr/lib/python3.4/vendor-packages/borgbackup-$(COMPONENT_VERSION)-py3.4.egg-info/requires.txt
 file path=usr/lib/python3.4/vendor-packages/borgbackup-$(COMPONENT_VERSION)-py3.4.egg-info/top_level.txt
+file path=usr/share/bash-completion/completions/borg
```

Comment:
This file should be in something like borg-common or borg-completion. BTW, can we should add borg-35...

Reply:
I added 3.5 version of `borgbackup`, but dumped the Bash completion as I was unable to add sub-packages.

Reply:
You can add necessary content (only content, no set actions) to borg-GENFRAG.p5m, look for example on gst-python/gst-python-GENFRAG.p5m